### PR TITLE
Versa Director CVE-2024-42450 and fixing version ranges, acknowledgments, section headings

### PR DESCRIPTION
**Pull request by amanion-cisa**
Adding Versa Director CVE-2024-42450 and fixing version ranges, acknowledgments, section headings for existing files.

### CODE REVIEW

**Review by justmurphy — APPROVED** [review]
lgtm

File: csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json [hunk 1]
```diff
@@ -9,31 +9,59 @@
...
           "schema": "https://docs.oasis-open.org/csaf/csaf/v2.0/csaf_json_schema.json",
           "version": "2.0"
         },
-        "id": "VA-24-317-01",
+        "id": "VA-24-325-01",
         "link": [
           {
-            "href": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-317-01.json",
+            "href": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-325-01.json",
             "rel": "self"
```
Comment (mstrad) [inline csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json]: This should be **"ref": "self"**

File: csaf_files/IT/white/2024/va-24-325-01.json [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        }
+      ],
+      "acknowledgments": [
+        {
+          "names": [
+            "[notnotnotveg](https://wiki.notveg.ninja/whoami/)"
+          ]
+        }
+      ],
+      "release_date": "2024-11-14T00:00:00"
```
Comment (mstrad) [inline csaf_files/IT/white/2024/va-24-325-01.json]: The date-time format recommends including the "Z" at the end of the timestamp.

**Review by mstrad — COMMENTED** [review]
Left a couple comments.

File: csaf_files/IT/white/2024/va-24-325-01.json (hunk 2 above)
Comment (justmurphy) [reply csaf_files/IT/white/2024/va-24-325-01.json]: Done.

File: csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json (hunk 1 above)
Comment (justmurphy) [reply csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json]: Done.

Comment (amanion-cisa) [reply csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json]: [CSAF 2.0 7.1.15](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#7115-requirement-15-rolie-feed) indicates "rel" which I think matches the [RFC](https://www.rfc-editor.org/rfc/rfc8322.html) also.

File: csaf_files/IT/white/2024/va-24-325-01.json (hunk 2 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-325-01.json]: I checked and some of the older files are mising the 'Z' on release_date. Editing the JSON files in the PR will break the signatures and hashes so I'll update the PR.

File: csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json (hunk 1 above)
Comment (justmurphy) [reply csaf_files/IT/white/cisa-csaf-it-feed-tlp-white.json]: @mstrad it looks like Art is correct. Should be "rel"
https://docs.oasis-open.org/csaf/csaf/v2.0/csd02/csaf-v2.0-csd02.html#7113-requirement-13-changescsv

**Pull request merged**